controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -150,7 +151,7 @@ private boolean isRemoveExisting() {
      *                                    to the configuration history directory
      */
     public ConfigurationFile(final File configurationDir, final String rawName, final String name, final boolean persistOriginal) {
-        this(configurationDir, rawName, name, persistOriginal ? InteractionPolicy.STANDARD : InteractionPolicy.READ_ONLY, false);
+        this(configurationDir, rawName, name, persistOriginal ? InteractionPolicy.STANDARD : InteractionPolicy.READ_ONLY, false, null, (ConfigurationExtension) null);
```

Comment:
Why do you need to cast the last `null` parameter?

Reply:
Not required